### ISSUE TITLE
wwwadmin doesn't show pet samples as associated to participants

### ISSUE THREAD

**Issue by wasade** [qiime/qiime_web_app#560]
See sample 000012213, which is associated to a pet survey in the web interface but doesn't appear to be associated to a survey in wwwadmin

**Comment by adamrp** [comment]
Are you talking specifically about Edit Participant, or are you noticing
the problem elsewhere, too?

**Comment by wasade** [comment]
check that sample on wwwadmin, the pet survey doesn't show up

**Comment by adamrp** [comment]
Do you mean when you do a search?  Or when you click edit participant?
 What are you trying to do?

**Comment by adamrp** [comment]
@wasade, note that editing of pet surveys is not currently supported anywhere (either by the user or on wwwadmin) and requires access to the database.  Is that what you were aiming to do here?

**Comment by wasade** [comment]
No. On wwwadmin, the sample 000012213 is associated with the pet Borte but
Borte does not appear to have an associated survey according to the
interface

**Comment by adamrp** [comment]
What part of the interface are you talking about?

**Comment by wasade** [comment]
The details for the kit that lists the kit ids, barcodes, etc. Each human
participant that has an associated survey is listed at the top. There is no
indication that "Borte" has any association with a valid survey. What would
be fantastic is if there was an indication that there was an actual survey
associated with the pet.
